views/pages/view-result.html

```diff
@@ -114,11 +114,11 @@ <h4>{{data.title}}</h4>
         &nbsp; &nbsp;
 
         {{#if data.media.audio}}
-        <a href="{{data.media.audio.urlProxied}}" class="btn btn-sm btn-default" target="_blank">Open audio in a new tab</a>
+        <a href="{{audioURL}}" class="btn btn-sm btn-default" target="_blank">{{audioContext}}</a>
```

Comment:
Basically the necessary change here is `{{data.media.audio.urlProxied}}` to `{{data.media.audio.urlProxied}}.{{data.media.audio.ext}}`.

No need to do the assignments in the route handler and this way we will have more consistent approach to generating these links.

Reply:
ok Tomorrow I'll do all changes 😄